### PR TITLE
[feat]: Added text utils

### PR DESCRIPTION
**Pull request by aryan-13**
-Added heading styles
-Added paragraph styles
-added text alignment classes

Preview: 
type

### CODE REVIEW

File: components/typography.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.heading-1,.heading-2,.heading-3,.heading-4,.heading-5{
+    padding:0;
+    font-weight:400;
+    margin:0;
+}
```
Comment (prachi-sahani) [inline components/typography.css:5]: instead, you can reset the styling of h1,h2..h6 tags in reset.css file

File: components/typography.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    text-align:center;
+    box-sizing:border-box;
+}
+.text-left{
+    text-align:left;
+    box-sizing:border-box;
+
+}
+.text-right{
+    box-sizing:border-box;
```
Comment (prachi-sahani) [inline components/typography.css]: repetitive styling can be moved to one common class or maybe you can use comma-separated selectors

File: documentation.html [hunk 3]
```diff
@@ -469,7 +469,7 @@ <h1 class="heading--1-medium">Code</h1>
 				<div class="main__about">
 					<h1 class="heading--1-medium">Sizes</h1>
```
Comment (prachi-sahani) [inline documentation.html:470]: IMO, you could have re-used your component library typography classes for documentation

File: components/typography.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+.p-md{
+    font-size: 2rem;
+}
+.p-lg{
+    font-size: 2.5rem;
+}
+/* Text ALIGNMENT */
+.text-centered{
+    text-align:center;
+    box-sizing:border-box;
```
Comment (stutisk) [inline components/typography.css]: you can make a seperate class for box-sizing ,as its common between three

**Review by stutisk — COMMENTED** [review]
else code looks good to me

File: components/typography.css (hunk 4 above)
Comment (aryan-13) [reply components/typography.css]: Done

File: documentation.html (hunk 3 above)
Comment (aryan-13) [reply documentation.html:470]: Yes, I'll change it

File: components/typography.css (hunk 2 above)
Comment (aryan-13) [reply components/typography.css]: Yes, Done.

File: components/typography.css (hunk 1 above)
Comment (aryan-13) [reply components/typography.css:5]: I'll update it

**Pull request merged**